src/style.css

```diff
@@ -0,0 +1,85 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.box {
+  display: flex;
+  flex-grow: 1;
+  height: 300px;
+  min-width: 300px;
+  justify-content: center;
+  align-items: center;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: white;
+}
+
+.box:nth-child(1) {
+  background-color: rgb(100%, 0%, 0%);
+  order: 1;
+}
+
+.box:nth-child(2) {
+  background-color: rgb(80%, 0%, 0%);
+  order: 2;
+}
+
+.box:nth-child(3) {
+  background-color: rgb(60%, 0%, 0%);
+  order: 3;
+}
+
+.box:nth-child(4) {
+  background-color: rgb(40%, 0%, 0%);
+  order: 4;
+}
+
+.box:nth-child(5) {
+  background-color: rgb(20%, 0%, 0%);
+  order: 5;
+}
+
+.box:nth-child(6) {
+  background-color: rgb(0%, 0%, 0%);
+  order: 6;

```

Comment:
Will something change If you delete orders for elements? Don't add unnecessary style